js_modules/dagster-ui/packages/ui-core/src/assets/AutoMaterializePolicyPage/AutomaterializeRightPanel.tsx

```diff
@@ -197,7 +197,6 @@ export const GET_POLICY_INFO_QUERY = gql`
         }
         autoMaterializePolicy {
           policyType
-          maxMaterializationsPerMinute
```

Comment:
this field is not actually being used, and now this information can be fetched from the rule